temba/channels/types/instagram/type.py

```diff
@@ -0,0 +1,65 @@
+import requests
+
+from django.conf.urls import url
```

Comment:
I know this was just copied from another channel but we've now updated the other channels - should use `from django.urls import re_path` instead. We're cleaning up use of deprecated functions so we can update to Django 4.